frontend/src/Editor/SaveAndPreview.jsx

```diff
@@ -52,18 +52,18 @@ class SaveAndPreview extends React.Component {
   };
 
   saveVersion = (versionId) => {
-    this.setState({ isSaving: true });
+    this.setState({ isSaving: versionId });
```

Comment:
the name `isSaving` signifies a boolean, can we change the name of the variable to something like `savingVersionId` or `savingVersion`?

Same suggestion for `isDeploying`

Reply:
Sure will make the change right away @Navaneeth-pk